### PR TITLE
docs: list path prefixes

### PR DESCRIPTION
**Pull request by eliobischof**
A compact list of ZITADEL path prefixes helps users configure reverse proxies or other networking components.

For example: https://discord.com/channels/927474939156643850/928175516664602694/1017101014060503080

### CODE REVIEW

File: docs/docs/apis/introduction.mdx [hunk 1]
```diff
@@ -224,3 +224,26 @@ Be aware that issuer, api, accounts and console domains do not exist anymore.
...
+/admin/
+/zitadel.system.v1.SystemService/
+/system/
+/saml/
+/assets/
+/ui/
+/oauth/v2/
+/oidc/
+/.well-known/openid-configuration
+/openapi/
```
Comment (livio-a) [inline docs/docs/apis/introduction.mdx:248]: did you omit the versions (apart from oauth) on purpose?

Comment (eliobischof) [reply docs/docs/apis/introduction.mdx:248]: Yes, so we can release new API versions without breaking proxy configs. Do you have a different opinion?

Comment (livio-a) [reply docs/docs/apis/introduction.mdx:248]: but the grpc endpoints will change then anyway

Comment (eliobischof) [reply docs/docs/apis/introduction.mdx]: :+1:  I added the version numbers

**Review by livio-a — APPROVED** [review]

**Pull request merged**